test/src/module/config/protocolTest.c

```diff
@@ -37,8 +37,13 @@ testRun(void)
                 harnessCfgLoad(cfgCmdArchiveGet, argList);
 
                 ProtocolServer *server = protocolServerNew(strNew("test"), strNew("config"), read, write);
-                protocolServerHandlerAdd(server, configProtocol);
-                protocolServerProcess(server, NULL);
+
+                static const ProtocolServerHandler commandHandler[] =
+                {
+                    {.command = PROTOCOL_COMMAND_CONFIG_OPTION, .handler = configOptionProtocol},
```

Comment:
I'm not sure I'm enamored with the rename of configProtocol to configOptionProtocol, but I guess it will have to do - but can you please rename it in line 21 here since we no longer have configProtocol()?

Reply:
Well, this `configProtocol -> configOptionProtocol` was to make it match all the other protocol functions, so I think that is fine. Fixed the test title, though, in 127aaf348a57.

What I did not notice is that we also have `configOptionProtocol` which is very confusing. I have renamed this to `configOptionRemote` in 127aaf348a57. Better?

Reply:
Exactly why I didn't like the name, so the rename looks good except   line 21 of config/protocolTest.c   
`if (testBegin("configOptionProtocol() and configProtocolOption()"))`

Reply:
Already fixed in cf663563c2160461 but I didn't link it before. I noticed these after fixed the others.